src/components/CodePreview/index.tsx

```diff
@@ -1,6 +1,7 @@
 import Highlight, { defaultProps } from "prism-react-renderer";
-import React, { CSSProperties, ReactNode } from "react";
-
+import React, { CSSProperties, ReactNode, useState, useEffect } from "react";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import startCase from "lodash/startCase";
```

Comment:
```suggestion
import startCase from 'lodash.startcase';
```
Could this not be imported this way so you don't need to disable the eslint rule? Would need the types installed too